src/app/Fake.Api.GitHub/GitHub.fs

```diff
@@ -104,91 +118,126 @@ module GitHub =
             return github
         }
 
-    let private makeRelease draft owner project version prerelease (notes:seq<string>) (client : Async<GitHubClient>) =
+    /// Creates a GitHub Release for the specified repository and tag name
+    /// ## Parameters
+    /// - `owner` - the repository's owner
+    /// - `repoName` - the repository's name
+    /// - `tagName` - the name of the tag to use for this release
+    /// - `setParams` - function used to override the default release parameters
+    /// - `client` - GitHub API v3 client
+    let CreateRelease owner repoName tagName setParams (client : Async<GitHubClient>) =    
```

Comment:
question is if we should add `owner`  `repoName` and `tagName` to the record as well. It "might" help to provide same script internal defaults if you create releases to multiple repos with the same owner.

I don't have a strong opinion on that, just something that crosses my mind. Question is if wen provide reasonable defaults for any of those parameter (but we already have APIs which have mandatory arguments in the record without default)

Reply:
I had considered, but thought as we had no defaults & they were required that it was better to explicitly call out they were required. I suppose it's nice that you could do `CreateRelease "owner" "repo" "v1.0" id ...` and it would just work.